mempcpy.c

```diff
@@ -2,7 +2,7 @@
  *
  */
 
-/* Written by Niels M�ller <nisse@lysator.liu.se>
```

Comment:
Does MSVC seriously complain about this? It's in a comment even :(

Out of these functions, which aren't available on Windows?

I'd almost prefer to remove any that aren't necessary on modern platforms.